crates/server/Cargo.toml

```
@@ -50,3 +50,7 @@ uuid = { workspace = true, features = ["v4", "serde"] }
 edgee-path.workspace = true
 edgee-sdk.workspace = true
 edgee-wasmtime.workspace = true
+
+[dev-dependencies]
+cargo-llvm-cov = "0.6.15"
+pretty_assertions = "1.4.1"
```

You need to use the `workspace` key here to use the workspace-defined dependencies :)

```suggestion
cargo-llvm-cov.workspace = true
pretty_assertions.workspace = true
```